.github/workflows/build-deploy.yml

```diff
@@ -58,7 +58,7 @@ jobs:
 
     deploy-to-labs-gcp:
         name: Deploy to labs-gcp
-        if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/<helst_master>'
+        if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/master'
```

Comment:
Kanskje vi kan droppe den endringen. Vil huske at `helst_master` betyr at det er mulig å være noe annet for en kort periode for test skyld. Litt usikker om det blir tolket som unødvendig repetisjon i kode om vi skriver `heads/master` 2 ganger 🤔